mountpoint-s3/src/main.rs

```diff
@@ -149,6 +149,9 @@ struct CliArgs {
     #[clap(long, help = "Set the 'x-amz-request-payer' to 'requester' on S3 requests", help_heading = BUCKET_OPTIONS_HEADER)]
     pub requester_pays: bool,
 
+    #[clap(long, help = "Set the storage class", help_heading = BUCKET_OPTIONS_HEADER)]
```

Comment:
We should display the default here.

Reply:
And "for new objects" in the help

Reply:
In case you missed it (since we do set defaults in a different way for some arguments): we can provide the default to Clap directly, and then we document and set the default value in one place.

We do the same for `part_size`: https://github.com/awslabs/mountpoint-s3/blob/ed542e726de9232f8cfbdbeeb3a5a37988ada891/mountpoint-s3/src/main.rs#L243-L250